Jenkinsfile

```diff
@@ -9,41 +9,44 @@ node {
   checkout scm
 
   stage 'Build'
-
-  /* Call the maven build. */
+  /* Call the Gradle build. */
   gradlew "build"
 
   /* Save Results. */
   stage 'Results'
 
   /* Archive the test results */
-  step([$class: 'JUnitResultArchiver', testResults: '**/build/test-results/**/TEST-*.xml'])
-
+  junit '**/build/test-results/**/TEST-*.xml'
   /* Archive the build artifacts */
-  step([$class: 'ArtifactArchiver', artifacts: 'build/lib/*.hpi,build/lib/*.jpi'])
+  archiveArtifacts artifacts: 'build/lib/*.hpi,build/lib/*.jpi'
 }
 
-/* Run maven from tool "mvn" */
+/* Run Gradle with the appropriate JDK */
 void gradlew(def args) {
   /* Get jdk tool. */
   String jdktool = tool name: "jdk7", type: 'hudson.model.JDK'
 
   /* Set JAVA_HOME, and special PATH variables. */
   List javaEnv = [
-    "PATH+JDK=${jdktool}/bin", "JAVA_HOME=${jdktool}",
+    "PATH+JDK=${jdktool}/bin",
+    "JAVA_HOME=${jdktool}",
     // Additional variables needed by tests on machines
     // that don't have global git user.name and user.email configured.
-    'GIT_COMMITTER_EMAIL=me@hatescake.com','GIT_COMMITTER_NAME=Hates','GIT_AUTHOR_NAME=Cake','GIT_AUTHOR_EMAIL=hates@cake.com', 'LOGNAME=hatescake'
```

Comment:
I also use them to deal with [Git inside of Docker](https://github.com/jenkins-infra/jenkins.io/blob/master/Jenkinsfile#L55-L62) :smiley_cat: 

I might change this to just use Docker instead of tools like it is now, since Docker allows for more developer flexibility here